2696. Minimum String Length After Removing Substrings/solution.py

```diff
@@ -1,11 +1,36 @@
 # Minimum String Length After Removing Substrings
 
-# this method takes in a string s with length between 1 and 100 consisting of only uppercase English letters, removes any instances of "AB" or "CD", and returns the length of the resulting sting
+# this method takes in a string s with length between 1 and 100 consisting of only uppercase English letters.
+# it removes any instances of "AB" or "CD", and returns the length of the resulting sting
+# removing one instance of "AB" or "CD" can create a new instance that needs to be removed
 
-# declare a variable strlen for the string length outside of the loop but use it as the limit for the loop
-# loop through the string while incrementing by 1 with strlen - 1 as the ending
-	# inner loop through the whole string, check if each index + the next letter is equal to AB
-	# second inner loop check if each index + the next letter is equal to CD
-		# in both loops, if a match is found, cut those two characters out of the string and reform as a new string
-			# if this happens decrement strlen by 2
-  # return strlen after the loops have finished
+class Solution(object):
+	def minLength(self, s):
+		
+	# declare a variable strlen for the string length outside of the loop and use it as the limit for the while
+	strlen = len(s)
+	
+	# loop through the string
+	i = 0
+	while i < strlen:
+		# check if each index + the next letter is equal to AB
+		if s[i:i+2] == "AB":
```

Comment:
using an "or" operator would let you not repeat the code: if s[i:i+2] == "AB" or s[i:i+2] == "CD"